prow/jobs/incubator/reconciler/reconciler-k3d.yaml

```diff
@@ -0,0 +1,297 @@
+                cpu: 2
+                memory: 4Gi
+    - name: pre-main-reconciler-cli-integration-kyma-latest-k3d
+      annotations:
+        owner: "jellyfish"
+        description: "int test reconciler"
+        pipeline.platform: "k3d"
+        pipeline.trigger: "pr-submit"
+      labels:
+        prow.k8s.io/pubsub.project: "sap-kyma-prow"
+        prow.k8s.io/pubsub.runID: "pre-main-reconciler-cli-integration-kyma-latest-k3d"
+        prow.k8s.io/pubsub.topic: "prowjobs"
+        preset-build-pr: "true"
+        preset-kind-volume-mounts: "true"
+        preset-dind-enabled: "true"
+        preset-kyma-guard-bot-github-token: "true"
+      run_if_changed: '^((cmd\S+|configs\S+|internal\S+|pkg\S+)(\.[^.][^.][^.]+$|\.[^.][^dD]$|\.[^mM][^.]$|\.[^.]$|/[^.]+$))'
+      skip_report: false
+      decorate: true
+      cluster: untrusted-workload
+      max_concurrency: 10
+      branches:
+        - ^main$
+      extra_refs:
+        - org: kyma-project
+          repo: kyma
+          path_alias: github.com/kyma-project/kyma
+          base_ref: main
+        - org: kyma-project
+          repo: test-infra
+          path_alias: github.com/kyma-project/test-infra
+          base_ref: main
+      spec:
+        containers:
+          - image: europe-docker.pkg.dev/kyma-project/prod/testimages/e2e-dind-k3d:v20230612-dadc01a7
+            command: ["/init.sh"]
+            securityContext:
+              privileged: true
+              seccompProfile:
+                type: Unconfined
+              allowPrivilegeEscalation: true
+            args:
+              - "bash"
+              - "-c"
+              - |
+                set -e
+                apk add nodejs npm
+                kyma_get_last_release_version_return_version=$(curl --silent --fail --show-error -H "Authorization: token ${BOT_GITHUB_TOKEN}" "https://api.github.com/repos/kyma-project/kyma/releases" | jq -r 'del( .[] | select( (.prerelease == true) or (.draft == true) )) | sort_by(.tag_name | split(".") | map(tonumber)) | .[-1].target_commitish | split("/") | .[-1]')
+                export KYMA_SOURCE="${kyma_get_last_release_version_return_version:?}"
+                install_dir="/usr/local/bin"
+                mkdir -p "$install_dir"
+                kyma_cli_url="https://storage.googleapis.com/kyma-cli-pr/kyma-linux-pr-${PULL_NUMBER}"
+                pushd "$install_dir" || exit
+                curl -Lo kyma "${kyma_cli_url}"
+                chmod +x kyma
+                popd || exit
+                kyma version --client
+                kyma provision k3d --ci
+                kyma deploy --ci --concurrency=8 --profile=evaluation --source="${KYMA_SOURCE}"
+                pushd "/home/prow/go/src/github.com/kyma-project/kyma/tests/fast-integration"
+                git remote add origin https://github.com/kyma-project/kyma.git
+                git reset --hard && git remote update && git fetch --all >/dev/null 2>&1 && git checkout "${KYMA_SOURCE}"
```

Comment:
Im thinking if isn't it just better to define the kyma version explicitly in the base_ref of extra_refs. Then this part would simply go down to:
```
kyma deploy --ci --profile=evaluation --source=local
make -C tests/fast-integration ci
```

Reply:
@Ressetkk But the kyma version here is fetched from the releases API (to get the latest release), how can I add the base_ref as the latest release in this case?